client/src/App.tsx

```diff
@@ -1,5 +1,7 @@
+// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
```

Comment:
Typescript can automatically define return type. Don't see a necessity for the explicit return type

Reply:
This rule enabling error if you don't define return type, not warning. Most rules already enabling with recommended syntax, but with warning. These configurations for enabling error, if you forgot setting return type. With errors lesser chances for passing eslint tests in commits, especially for "new-bies"

Reply:
I understand that it throws an error) I say that I don't see the necessity for this rule. I suggest turning this rule off.

Reply:
Most rules which we're using for our understading of code structure, by under hood there are already can be integrated

Reply:
Steel don't think that this is a necessary rule...

Reply:
Doesn't everyone understanding typescript in our project :) this is improving learning and understanding of typescript